### ISSUE TITLE
Hello!! Could you develope a Buy and Sell script in Kotlin for me?

### ISSUE THREAD

**Issue by ronaldoguedess** [religion-counter/onlyone#14]
I could pay for that!  I'm developing my APP and I need this feature in Kotlin using web3j.

I'm waiting your answer.

**Comment by religion-counter** [comment]
Hi @ronaldoguedess ,
I am developing in Java, Python and TypeScript. But for web3 interactions I prefer Java and TypeScript.
Actually I will implement Buy and Sell for Onlyone in Java and Typescript in this github repo and will make them publicly available together with the whole site and casino. Unfortunately lately I don't have much time to do this as I am doing it in my spare time, but soon I will post what you need in this repo.
If someone else can help you faster, good luck!

Regards,

**Comment by ronaldoguedess** [comment]
Nice bro!! JAVA works for me as well... it will be amazing and help me a lot to create a direct swap function on my App.
👏👏👏  Do you have a forecast that when it will be done?

**Comment by religion-counter** [comment]
If you need similar functionality like in https://pancakeswap.finance/swap I think that I will add swap with BNB/BUSD/ONLYONE in onlyonefinance.com similar to pancakeswap.finance in 2-3 weeks. It will not have very good UI and will be for testing purposes only and it will use the pancakeswap router smart contract.

**Comment by ronaldoguedess** [comment]
Yep... it's exactly what I need!  thanks. I will wait to see that! Could you start developing that?
Don't worry about the layout, the important thing is the backend works. haha

Before I forget, congratulations on your crypto project. Nice supply hahaha, just 1 tokens with a big burn! 
I also have my own... [ecoin-finance](https://ecoin-finance.com)

**Comment by ronaldoguedess** [comment]
Hello bro.. Did you started the Swap feature?

**Comment by religion-counter** [comment]
Hi @ronaldoguedess ,
Sorry for the late response. Congratulations for the great project ecoin. I hope you have great future with it. I still haven't started working seriously on my #onlyonecasino, but I will post all my updates in this github repo. I plan to just make transactions like in the script but with Web3 Java code.

**Comment by ronaldoguedess** [comment]
Thanks so much my friend!! Sorry for my delay. Working busy all the time also. haha . Do you have news?
